### ISSUE TITLE
Add documentation for actions.yaml

### ISSUE THREAD

**Issue by LeviPesin** [ossf/allstar#324]
It is not very clear from the current documentation how to configure actions.yaml check and what are potential examples and use-cases of it.

**Comment by agonzalez-plume** [comment]
Yeah, I requested something similar. Didn't yet get actions.yaml working, but the project wanted help in creating an examples directory

This other Issue has some on helping with documentation: https://github.com/ossf/allstar/issues/316

**Comment by pburkholder** [comment]
I've not see actions.yaml working, so it may be an implementation bug.
